### ISSUE TITLE
Game UI gets clipped if the window is too small

### ISSUE THREAD

**Issue by jalejos** [mayela/rock-paper-scissors#2]
The game UI is not shown properly on smaller windows. Maybe we could add Flexbox to display all of the elements (might need to change some of the images' properties as well so they keep their aspect ratio)

**Comment by mayela** [comment]
Yup, this is one of the most important improvements to do. Thanks
